src/quantize_linear.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+import {add, div} from './binary.js';
+import {clamp} from './clamp.js';
+import {unary} from './unary.js';
+
+function roundToNearestEvens(x) {
+  return Math.floor(x) % 2 == 0 ? Math.floor(x) : Math.ceil(x);
+}
+
+/**
+ * Elementwise operator to quantize floating point down to a low precision integer
+ * (typically uint8 with a zero-point bias).
+ * The calculation follows the expression roundToNearestEvens(input / scale) + zeroPoint.
+ * @param {Tensor} input
+ * @param {Tensor} scale
+ * @param {Tensor} zeroPoint
+ * @param {String} dataType
+ * @return {Tensor}
+ */
+export function quantizeLinear(input, scale, zeroPoint, dataType) {
+  const addOutput = add(div(input, scale), zeroPoint);
+  const roundOutput = unary(addOutput, (x) => roundToNearestEvens(x));
```

Comment:
```suggestion
  const dividedOutput = div(input, scale);
  const roundedOutput = unary(dividedOutput, (x) => roundToNearestEvens(x));
  const addedOutput = add(roundedOutput, zeroPoint);
```
Round before adding.

Reply:
Updated, thanks.